.github/workflows/ping-other-repos.yml

```
@@ -0,0 +1,25 @@
+name: Remote Dispatch Action Initiator
```

I like how the file name described this and thought it would be nice if it would be visible in the GitHub UI

```suggestion
name: Ping other repos
```